CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This project makes use of the [Sementic Versioning](http://semver.org/)
 
 ### Added
 - Set the max db `pool` size via an ENV var called `DB_POOL_SIZE`
+- Fixed bug when redirect-domains were set on a site with SSL.
```

Comment:
Can you put this under a heading called `### Fixed`? (That way its easier for people that ran into this issue to spot if an issue has been solved for them :) )